base/shell/cmd/lang/ro-RO.rc

```diff
@@ -1,4 +1,10 @@
-/* Translator: Ștefan Fulea (stefan dot fulea at mail dot com) */
+/*
+ * PROJECT:     ReactOS Command Prompt
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
```

Comment:
the PURPOSE line is imho totally superfluous because line 9 tells us exactly what it is, and that without (potential) typos like ukraiaiaiaiainian. And with compile-time-checks even. The definition of a bloat-line in rcs.

Reply:
So, should I keep it or delete it? I did the same in every Romanian resource file. I undestood that these are the rules.

Reply:
Ok, I deleted it.

Reply:
Suggestion is incorrect, please keep this per standard.

Reply:
I understand. I'll revert the change.